linux/entrypoint/pre_start.sh

```diff
@@ -12,6 +12,7 @@ $ENTRYPOINT_HOME/certs.sh
 
 props set org.codice.ddf.system.hostname $_app_hostname $APP_HOME/etc/system.properties
 props set org.codice.ddf.system.siteName $_app_hostname $APP_HOME/etc/system.properties
+props set org.codice.ddf.system.httpsPort $APP_HTTPSPORT $APP_HOME/etc/system.properties
```

Comment:
minor request: could you change this variable to be `HTTPS_PORT` I have another set of changes going in for some more fine grained port control and they follow that pattern.